examples/SC_noise_capture.py

```diff
@@ -0,0 +1,96 @@
+import pyspecdata as ps
+import numpy as np
+import os
+from numpy import r_
+import SpinCore_pp as sc
+from datetime import datetime
+
+my_exp_type = "ODNP_NMR_comp/noise_tests"
+assert os.path.exists(ps.getDATADIR(exp_type=my_exp_type))
+config_dict = sc.configuration("active.ini")
+nScans = 100
+# {{{ add file saving parameers to config dict
+config_dict["chemical"] = (
+    config_dict["chemical"] + "_" + str(config_dict["SW_kHz"]) + "kHz"
+)
+config_dict["type"] = "noise"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["noise_counter"] += 1
+# }}}
+# {{{ SpinCore settings - these don't change
+tx_phases = r_[0.0, 90.0, 180.0, 270.0]
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = sc.get_integer_sampling_intervals(
+    SW_kHz=config_dict["SW_kHz"],
+    time_per_segment_ms=config_dict["acq_time_ms"],
+)
```

Comment:
in manuscript MWE we hardset these values but figure for this PR I incorporate this function like other examples